daily_read/__main__.py

```diff
@@ -64,22 +64,21 @@ def generate_all():
     orderer_with_modified_projects = projects_data.find_unique_orderers()
 
     op = daily_read.order_portal.OrderPortal(config_values, projects_data=projects_data)
-    op.get_orders()
+    for orderer in orderer_with_modified_projects:
+        if orderer:
+            op.get_orders(orderer=orderer)
     # TODO - not sure how to proceed, fetching ALL projects in order portal
     # and then generate reports seems quite slow and wasteful, but I haven't
     # tried so it might work. Maybe a pull model where we only generate reports
     # for projects that we know have been updated can be a better alternative?
-    all_sthlm_orders = op.process_orders(use_node="Stockholm")
+    modified_orders = op.process_orders()
     # TODO: Should clean up...
```

Comment:
Could you perhaps remove the comments that are no longer valid? Also goes for the TODO above, but that one might still be valid?

Reply:
I've removed the ones I think are done. The others are still valid.